spec/betterc.dd

```diff
@@ -0,0 +1,47 @@
+Ddoc
+
+$(SPEC_S Better C,
+
+`-betterC` is a Command-Line flag for `dmd`, which generates light-weight,
+Druntime-free binaries with are suitable for linking from C.
+
+---
+extern(C) void main()
+{
+    import core.stdc.stdio : printf;
+    printf("Hello betterC\n");
+}
+---
+
+$(CONSOLE
+> dmd -betterC hello.d && ./hello
+Hello betterC
+)
+
+$(H2 $(LNAME2 consequences, Consequences))
+
+As no DRuntime is available, many D features won't work.
+For example:
+
+$(OL
+    $(LI Garbage Collection)
+    $(LI Thread-local storage)
+    $(LI TypeInfo and ModuleInfo)
+    $(LI `extern(D)` Classes)
+    $(LI Built-in threading (e.g. $(MREF core, thread)))
+    $(LI Dynamic arrays (but not slices) and associative arrays)
+    $(LI Exceptions)
+    $(LI `switch` with strings)
+    $(LI `final switch`)
+    $(LI `synchronized` and $(MREF core, sync))
+    $(LI Static module constructors or deconstructors)
+    $(LI Struct deconstructors)
+    $(LI `unittest`)
```

Comment:
Since you're not linking druntime there would be no test runner to run the unittests. In theory you could write your own, though I don't know if that's feasible. Instead what I think we should explain is that you can use unittests to test the functions you're using in `-betterC` mode by building without `-betterC` your unittest builds.